manager/controllers/utils.go

```diff
@@ -29,15 +29,6 @@ var tagRegex = regexp.MustCompile(`([^/=]+)/([^/=]+)(=([^/=]+))?`)
 var enableCyndiTags = utils.GetBoolEnvOrDefault("ENABLE_CYNDI_TAGS", false)
 var disableCachedCounts = utils.GetBoolEnvOrDefault("DISABLE_CACHE_COUNTS", false)
 
-var validSystemProfileFilters = map[string]bool{
-	"sap_sids":                    true,
-	"sap_system":                  true,
-	"mssql":                       true,
-	"mssql->version":              true,
-	"ansible":                     true,
-	"ansible->controller_version": true,
-}
-
```

Comment:
don't we need this map to allow only certain keywords for filter? We should be able to return 400 if user tries something like `filter[system_profile][asdf]=asdf` but I don't know if we can do it with the old code or not

Reply:
It's checked against` nestedFilters` map
https://github.com/RedHatInsights/patchman-engine/pull/1277/files/fc656574644903bfb6a2aaba18ee18b3e12f91c2..47c4a56c8ea79e937dab87416df48749c0ba456d#diff-dca21f2371a54b52831672e4a621006fc040a2986ce17b3bfd408b8b3cc2452bL105-L117